specs/phase0/beacon-chain.md

```diff
@@ -241,18 +241,19 @@ The following values are (non-configurable) constants used throughout the specif
 
 ### Rewards and penalties
 
-| Name | Value |
-| - | - |
-| `BASE_REWARD_FACTOR` | `2**6` (= 64) |
-| `WHISTLEBLOWER_REWARD_QUOTIENT` | `2**9` (= 512) |
-| `PROPOSER_REWARD_QUOTIENT` | `2**3` (= 8) |
-| `INACTIVITY_PENALTY_QUOTIENT` | `2**25` (= 33,554,432) |
-| `MIN_SLASHING_PENALTY_QUOTIENT` | `2**5` (= 32) |
+| Name | Value | Description |
+| - | - | - |
+| `BASE_REWARD_FACTOR` | `2**6` (= 64) | Global scaling factor for all rewards (and for penalties that are proportional to those rewards). If total deposits are N gwei, then the theoretical max per-epoch reward rate is roughly `BASE_REWARD_FACTOR / sqrt(N)` (eg. 1m ETH deposited -> `10**15` gwei -> `64 / 10**7.5 ~= 0.000002024` per-epoch return -> 0.1663 annual return, as 1 year = `31556926 / 384` epochs)
+| `WHISTLEBLOWER_REWARD_QUOTIENT` | `2**9` (= 512) | If you catch someone and they get slashed, you get `1 / WHISTLEBLOWER_REWARD_QUOTIENT` of their balance. |
+| `PROPOSER_REWARD_QUOTIENT` | `2**3` (= 8) | If you include a message of certain types in your block as a proposer, you get `1 / PROPOSER_REWARD_QUOTIENT` of the reward that they get |
+| `INACTIVITY_PENALTY_QUOTIENT` | `2**25` (= 33,554,432) | If the chain has lost finality for N epochs, then each offline validator loses `N/INACTIVITY_PENALTY_QUOTIENT` of their balance during that epoch. When `N^2/2 = INACTIVITY_PENALTY_QUOTIENT`, the offline validatorsThe goal is to make it unambiguously not profitable to censor >1/3 lose `~1-1/e ~= 63.2%` of their balance. That happens after `2**13` epochs, or 36 days. |
```

Comment:
`INACTIVITY_PENALTY_QUOTIENT` has been fixed to `2**24` on [v012x branch ](https://github.com/ethereum/eth2.0-specs/blob/v012x/specs/phase0/beacon-chain.md#rewards-and-penalties)